src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObject = {};
+
+  for (const key in robot) {
+    if (inverseObject.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    inverseObject[robot[key]] = key;
+  }
```

Comment:
Value `robot[key]` was used 2 times.
Maybe it's better to put it in a variable?

Reply:
@33Orange thanks, it's good advice